nmdc_server/ingest/biosample.py

```diff
@@ -31,6 +31,10 @@ def extract_extras(cls, values):
                 lat, lon = values.pop("lat_lon")["has_raw_value"].split(" ")
                 values["latitude"] = float(lat)
                 values["longitude"] = float(lon)
+        else:
+            # Workaround/testing code -- not to be included in PR
```

Comment:
Should this code be in this PR? Either remove the else clause or the comment.

Reply:
d65287e